selfdrive/camerad/cameras/sensor2_i2c.h

```diff
@@ -110,8 +110,8 @@ struct i2c_random_wr_payload init_array_ar0231[] = {
   {0x31E0, 0x0003},
 
   // HDR Settings
-  {0x3082, 0x0004}, // OPERATION_MODE_CTRL
-  {0x3238, 0x0444}, // EXPOSURE_RATIO
+  {0x3082, 0x0004}, // OPERATION_MODE_CTRL (A) (enable HDR + 2 exposures)
+  {0x3238, 0x0004}, // EXPOSURE_RATIO
```

Comment:
Add comment that this is 16:1 exposure ratio?

Reply:
For now it's just this. Once our tone mapping is good enough to actually use all this dynamic range I'll see if we want to add a gain ratio too.